website/docs/how-to/how-to-add-sso-open-id-connect.md

```diff
@@ -76,3 +76,45 @@ Log out of Unleash and sign back in again. You should now be presented with the
+
+| Variable name      | Purpose                                                               | Required | Example values                                                |
+|--------------------|-----------------------------------------------------------------------|----------|---------------------------------------------------------------|
+| OIDC_ENABLED       | Tells Unleash to use environment variables for configuring OIDC       | yes      | true / false (false will turn off OIDC login)                 |
+| OIDC_DISCOVER_URL  | Tells Unleash how to configure OIDC                                   | yes      | https://myoidchost.azure.com/.well-known/openid-configuration | 
+| OIDC_CLIENT_ID     | The OIDC client ID of this application.                               | yes      | FB87266D-CDDB-4BCF-BB1F-8392FD0EDC1B                          |
+| OIDC_CLIENT_SECRET | Shared secret from OpenID server. Used to authenticate login requests | yes      | qjcVfeFjEfoYAF3AEsX2IMUWYuUzAbXO                              |
+
+Once these are configured OIDC should be working.
+
+### Step 2 (optional) - Auto create users from specific email domains
+
+| Variable name                  | Purpose                                                                                        | Required | Example values              |
+|--------------------------------|------------------------------------------------------------------------------------------------|----------|-----------------------------|
+| OIDC_AUTO_CREATE               | Tells Unleash to auto create users from the specific domains in OIDC_AUTO_CREATE_EMAIL_DOMAINS | no       | true / false                |
+| OIDC_AUTO_CREATE_EMAIL_DOMAINS | A comma separated list of domains to auto-create users for                                     | no       | getunleash.io,getunleash.ai | 
+
+### Step 3 (optional) Enable group syncing
+
+| Variable name               | Purpose                                                                                                  | Required   | Example values              |
+|-----------------------------|----------------------------------------------------------------------------------------------------------|------------|-----------------------------|
+| Variable name               | Purpose                                                                                                  | Required   | Possible values             |
+| --------------------------- | -------------------------------------------------------------------------------------------------------- | ---------- | -----------------           |
+| OIDC_ENABLE_GROUP_SYNCING   | Tell Unleash to setup group syncing from OIDC login requests                                             | No         | true or false               |
+| OIDC_GROUP_JSON_PATH        | a json path expression telling where in the response Unleash can find the group membership information   | No         | groups                      |
```

Comment:
Nit: for the example value, I'd try to use a more complex json path example, e.g. `org[0].groups` or something like that so users with more complex use cases know how to specify this

Reply:
I just copied this from our example that was already present.